examples/demo_coordinate_system_datasets.ipynb

```diff
@@ -0,0 +1,916 @@
+   "metadata": {},
+   "source": [
+    "# Tests of coordinate system effects on shear profiles\n",
+    "\n",
+    "Author:  Tomomi Sunayama, July 15, 2024\n",
+    "\n",
+    "Tested, modified, and documented by: Camille Avestruz, July 18, 2024\n",
+    "\n",
+    "\n",
+    "This notebook illustrates the impact of having an incorrect coordinate system when calculating shear profiles.  The default coordinate system for `CLMM` is the euclidean/pixel coordinate system.  This is consistent with DC2 catalogs.  However, if we input a catalog that assumes a celestial (sky) coordinate system and use the default euclidean (pixel) coordinate system (or vice versa), the signal of shear around a cluster disappears because the signal essentially looks random.\n",
+    "\n",
+    "We test:\n",
+    "- CosmoDC2 source galaxies with shears extracted from `TXPipe` for a single galaxy cluster (euclidean/pixel coordinate system)\n",
+    "- Example source galaxies for galaxy clusters from a [Summer School](https://github.com/oguri/wlcluster_tutorial) taught by Masamune Oguri (euclidean/pixel coordinate system)\n",
+    "- HSC Y3 source galaxies with shears post processed by Tomomi Sunayama (celestial/sky coordinate system)\n",
+    "\n",
+    "We also \n",
+    "- Compare the explicit calculation of a shear profile on the HSC Y3 source galaxies against a profile produced from `CLMM`. \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Instructions to download text data\n",
+    "\n",
+    "Before running this notebook, you will need to download some data.  The data is available through a [dropbox link](https://www.dropbox.com/scl/fo/dwsccslr5iwb7lnkf8jvx/AJkjgFeemUEHpHaZaHHqpAg?rlkey=efbtsr15mdrs3y6xsm7l48o0r&st=xb58ap0g&dl=0)\n",
+    "\n",
+    "First, create a directory where you want to put the example data, e.g. for a given `data_coords_dir`:\n",
+    "```\n",
+    "data_coords_dir=\"<YOUR PATH TO DATA COORDS DIR>/data_coords\"\n",
+    "mkdir -p $data_coords_dir\n",
+    "cd $data_coords_dir\n",
+    "```\n",
+    "Download all files from [dropbox link](https://www.dropbox.com/scl/fo/dwsccslr5iwb7lnkf8jvx/AJkjgFeemUEHpHaZaHHqpAg?rlkey=efbtsr15mdrs3y6xsm7l48o0r&st=xb58ap0g&dl=0).  This will be a zip file, `CLMM_data.zip` of size 242 Mb. scp or move this to `data_coords_dir`.\n",
+    "\n",
+    "From the directory, you should be able to unzip:\n",
+    "```\n",
+    "unzip data_CLMM.zip -d .\n",
+    "```\n",
+    "You now have the necessary data files to run this notebook.  \n",
+    "\n",
+    "**Make sure to change the `data_coords_dir` variable in the cell below to the appropriate location where you unzipped these files.**\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "#  CHANGE THIS TO YOUR LOCATION\n",
+    "data_coords_dir=\"/pscratch/sd/a/avestruz/clmm-area/CLMM_test_data/data_coords/\""
```

Comment:
Maybe use a mock path so the user knows it has to be changed?

Reply:
Ah, fair - you mean something like `<YOUR ROOT PATH TO>/data_coords/`?

Reply:
Done